beesdoo_stock/models/stock.py

```diff
@@ -1,28 +0,0 @@
-from odoo import api, models
-
-
-class StockPackOperation(models.Model):
-    _inherit = "stock.picking"
-
-    @api.multi
-    def actions_on_articles(self):
-        ids = self._ids
-        context = self._context
-        ctx = (context or {}).copy()
-        ctx["articles"] = []
-        for line in self.browse(ids).move_line_ids:
-            ctx["articles"].append(line.product_id.product_tmpl_id.id)
-        if ctx["articles"]:
-            return {
-                "name": "Articles",
-                "view_type": "list",
-                "view_mode": "list",
-                "res_model": "product.template",
-                "view_id": False,
-                "target": "current",
-                "type": "ir.actions.act_window",
-                "context": ctx,
-                "nodestroy": True,
-                "res_id": ctx["articles"],
-                "domain": [("id", "in", ctx["articles"])],
-            }
```

Comment:
I don't understand, this is not the same code as in https://github.com/OCA/stock-logistics-workflow/pull/1119/files#diff-df167430c1f8baf5c14a15db17f7fc03019412c41df18fee8042b289c605ffc2. Is it just a split or did you change some stuff ?

Reply:
I recoded it the proper way indeed.